From 6a0716993439d00eb4ebdaf47115e7bd4d97429e Mon Sep 17 00:00:00 2001
From: Kent Russell <kent.russell@amd.com>
Date: Thu, 27 Apr 2017 10:12:53 -0400
Subject: [PATCH 5587/5855] drm/amdkfd: Fix kernel compile warning

We know that props will be defined in the loop, but set it to NULL to
appease the compiler

Change-Id: Ie18c3b38be6e66a9af4b4642d6ebc126cee19ea0
Signed-off-by: Kent Russell <kent.russell@amd.com>
---
 drivers/gpu/drm/amd/amdkfd/kfd_crat.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_crat.c b/drivers/gpu/drm/amd/amdkfd/kfd_crat.c
index a928f45..e5a5574 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_crat.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_crat.c
@@ -303,7 +303,7 @@ static int kfd_parse_subtype_cache(struct crat_subtype_cache *cache,
 static int kfd_parse_subtype_iolink(struct crat_subtype_iolink *iolink,
 					struct list_head *device_list)
 {
-	struct kfd_iolink_properties *props, *props2;
+	struct kfd_iolink_properties *props = NULL, *props2;
 	struct kfd_topology_device *dev, *cpu_dev;
 	uint32_t id_from;
 	uint32_t id_to;
-- 
2.7.4

